lib/utils/cert-utils.js

```diff
@@ -0,0 +1,103 @@
+const os = require('os');
+const path = require('path');
+const fs = require('fs-extra');
+const forge = require('node-forge');
+const logger = require('@blackbaud/skyux-logger');
+
+const certCommonName = 'SKYUX-Developer-Certificate';
+const certName = 'skyux-server.pem';
+const keyName = 'skyux-server.key';
+
+const defaultCertRootPath = path.resolve(`${os.homedir()}/.skyux/certs/`);
+const defaultCertPath = path.join(defaultCertRootPath, certName);
+const defaultKeyPath = path.join(defaultCertRootPath, keyName);
+
+function error(msg) {
+  logger.error(msg);
+  return false;
+}
+
+function validate(argv) {
+  const cert = getCertPath(argv);
+  const key = getKeyPath(argv);
+
+  if (!fs.existsSync(cert)) {
+    return error(`Error locating certificate: ${cert}`);
```

Comment:
When I type `skyux certs untrust` and then `skyux certs validate`, I don't see this error message. Is that because it's seeing my "old" SKY UX certificate? Is there any way to clear the old one out too?

Reply:
It's not passing because it's finding your old cert, it's passing because those files still exist in `~\.skyux\certs`. I think the `TODO` a bit below is where that validation might occur.

Do we need that done now?

Reply:
Hmm... think `untrust` should also delete them?  I'm leaning towards yes.

Reply:
That was my initial thought as well since `trust` always generates a new cert.

Reply:
Added a `remove` method that `untrust` calls.

Reply:
Tested and it works!